migration/idempotent/013-apply-config.sql

```diff
@@ -81,6 +88,12 @@ BEGIN
         AND d.deptype = 'e'
         AND e.extname = 'promscale'
         AND k.relkind IN ('r', 'p')
+        AND (n.nspname, k.relname) NOT IN
+        (
+            -- these should NOT be config tables
+            ('_ps_catalog'::name, 'migration'::name),
```

Comment:
add 
```
--migration table will be populated by the installation of extension
```
and
```
--we want the telemetry data reset with restores
```

Reply:
added in db5c7579dd6c0d87d48cf6fd888ede3cd7ce2a67